pkgs/development/python-modules/mohawk/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi, python, mock, nose, pytest, six }:
+with lib;
+buildPythonPackage rec {
+  pname = "mohawk";
+  version = "1.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "08wppsv65yd0gdxy5zwq37yp6jmxakfz4a2yx5wwq2d222my786j";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  checkInputs = [ mock nose pytest ];
+
+  checkPhase = ''
+    pytest mohawk/tests.py
+  '';
+
+  meta = {
+    description = "Python library for Hawk HTTP authorization.";
+    homepage = "https://github.com/kumar303/mohawk";
+    license = licenses.mpl20;
+    maintainers = [ ];
```

Comment:
please add yourself as a maintainer :)

Reply:
I can do this, but just to be clear this PR is purely from ZHF, I don't use this package or Alerta. As I noted in the second paragraph I don't even know how to test importing this package currently.

Reply:
ah, then you can skip it... but it's just nice to have someone who has context

Reply:
Cool, I will skip for now. Any suggestions on the equivalent of `python3 -c "import mohawk"` but using mohawk from `result`? Best I've come up with is `PYTHONPATH=result/lib/python3.8/site-packages:/nix/store/md9ld5w5s4dkqkm0rjbsqrlzlhyjiq4y-python3.8-six-1.15.0/lib/python3.8/site-packages python -c "import mohawk"`.

Reply:
```
pythonImportsCheck = [ "mohawk" ];
```

Reply:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
That's helpful, although I more so mean an interactive Python REPL outside of the `.nix` file that can import from `result`. But I don't want this to turn into a support thread, and I think the PR is ready at this point.

Reply:
> Any suggestions on the equivalent of `python3 -c "import mohawk"` but using mohawk from `result`

```
nix-shell -I nixpkgs=$PWD -p python3Packages.mohawk --run "python -c 'import mohawk'"
```